app/src/main/java/com/cfirmo33/moviesapp/api/RetrofitUtils.java

```diff
@@ -0,0 +1,69 @@
+package com.cfirmo33.moviesapp.api;
+
+import com.cfirmo33.moviesapp.MainApplication;
+
+import java.security.cert.CertificateException;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by Carlos on 13/09/2016.
+ */
+public class RetrofitUtils {
+
+    private static Retrofit retrofit;
+
+    private static OkHttpClient getUnsafeOkHttpClient() {
```

Comment:
This is interesting question, that is a class that i use in my job, why we have an internal network with an improperly configured proxy server and use this approach to be able to access the external network without report a
authentication, but I agree with you it may be just for my isolated case.